opscenter/README.md

```diff
@@ -0,0 +1,89 @@
+Setting up an OpsCenter
+===
+An OpsCenter is a service that you can setup to provide a distribution point for your applications and dependant applications. Depending on how you choose to configure the service, it can retrieve updates from Gravitational's OpsCenter.
+
+Pre-requisites
+---
+ - Login for [https://get.gravitational.io](https://get.gravitational.io)
+ - Telekube binaries
+
+Create an SSL certificate
+---
+The OpsCenter will use SSL to secure connections to it. In this repository are the files `example-server.crt` and `example-server.key` which hold the certificate and key to a self-signed certificate for `example.gravitational.io`. Do deploy your own OpsCenter, you should create a certificate for your server. We recommend using a certificate signed by a trusted root authority, but you can use self-signed certificates if you wish. Here's  some information on [how to create your own self-signed certificate](http://www.akadia.com/services/ssh_test_certificate.html).
+
+Once you have a certificate, please place the PEM encoded certificate in this directory as `server.crt` and they PEM encoded key as `server.key`.
+
+Generating a token
+---
+To connect your OpsCenter to `get.gravitational.io` to retrieve updates, you'll need an API token. Using the `tele` tool, you can quickly get one. First, login:
+
+```
+$ tele login -o get.gravitational.io
+If browser window does not open automatically, open it by clicking on the link:
+ https://accounts.google.com/o/oauth2/v2/auth?client_id=2...snip
+Ops Center:	get.gravitational.io
+Username:	meta@gravitational.io
+Cluster:	N/A
+Expires:	Wed Mar 29 11:48 UTC (19 hours from now)
+```
+
+Finally, using your authenticated session, generate an API token:
+
+```
+$ tele keys new
+4340d6dbf469c13ba9ff56efccddffeb0ac1faf107fe99e49aa668ec85ec4e99%
+```
+
+We'll need to use this value in an environment variable named `TOKEN`, but for now you can store it in a file, or wherever is convenient.
+
+Configuring OpsCenter
+---
+The last step prior to provisioning is creating a file with the full set of configuration. We've provided an example configuration for you in the file `example-opscenter.yaml`.
+
+```
+advertise_addr: <example.gravitational.io:443>
+ssh_advertise_addr: <example.gravitational.io:33008>
+tls:
+  cert_file: /etc/gravitational/server.crt
+  key_file: /etc/gravitational/server.key
+oidc_connectors:
+  - id: google
+    redirect_url: https://<example.gravitational.io>/portalapi/v1/oidc/callback
+    client_id: <example>.apps.googleusercontent.com
+    client_secret: <secret>
+    issuer_url: https://accounts.google.com
+smtp:
+  enabled: true
+  password: <pass>
+  username: username
+  server: smtp.mailgun.org:587
+users:
+  # list of static users pre-configured for this portal
+  - email: "admin@yourdomain.com"
+    password: please-generate-it
+    type: "admin"
+    org: "yourdomain.com"
+    identities:
```

Comment:
Of course now is late but after going through the exercise of setting up my ops center, its not clear the configuration of the connector:
```yaml
users:
  # list of static users pre-configured for this portal
  - email: "admin@yourdomain.com"
    password: please-generate-it
    type: "admin"
    org: "yourdomain.com"
    identities:
      - connector_id: "google"
        email: "admin@yourdomain.com"
```

```yaml
users:
  - email: "admin@yourdomain.com"
    password: please-generate-it
```
You have there `email:` && `password:` --> what are these use for? Apparently nothing :( after installing... maybe because of a bug but if these were documented then the one documenting it would have tested them at least.

```yaml
users:
    identities:
      - connector_id: "google"
        email: "admin@yourdomain.com"
```

This `email:` is the only real configuration and has to be set to the email of the user that is going to be able to login and that will always be admin. This is worth documenting for people to understand how to configure this section.